3-networks-hub-and-spoke/modules/base_env/versions.tf

```diff
@@ -19,11 +19,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.50"
+      version = ">= 3.50, != 6.26.0"
```

Comment:
@renato-rudnicki @mariammartins 
it needs to be `< 6.26.0`
here and every where until the issue is fixed.

please also add a comment with the link to the issue in the provider repository